### DISCUSSION TITLE
Key press after wake up from suspend

### DISCUSSION THREAD

**Discussion by lifefir3** [kovidgoyal/kitty#5413] [General]
Hi Guys

System: Ubuntu 22.04 with Wayland
Kitty:      0.25.2

Does anyone has key press and hold problem after waking up from suspend?
After waking up from suspend, I only get a single respond by holding any key down, but 
pressing and releasing  a key works.
  
Also relaunching kitty will fix the problem.I was trying to see if I get any error message 
by running a second kitty from the first, but did get any error message.

Thanks

**Comment by page-down** [comment]
You can try the nightly release.

https://github.com/kovidgoyal/kitty/issues/5188

**Comment by lifefir3** [comment]
Thank you, it does fix the problem

**Comment by kovidgoyal** [comment]
dup of #5188
